sw/device/tests/otbn_irq_test.c

```diff
@@ -3,13 +3,12 @@
 // SPDX-License-Identifier: Apache-2.0
 
```

Comment:
nit: maybe this file should be name `otbn_ecdsa_op_irq_test.c` to stay consistent with the testpoint name and in the case that additional OTBN tests are added that also fire an interrupt.